app/models/theme.rb

```diff
@@ -15,12 +15,7 @@ class Theme < ActiveRecord::Base
 
   validates :title, presence: {message: _("can't be blank")}
 
-  # EVALUATE CLASS AND INSTANCE METHODS BELOW
-  #
-  # What do they do? do they do it efficiently, and do we need them?
-
-
-
+  scope :updated_at_desc, -> { self.all.order(updated_at: 'DESC') }
```

Comment:
Don't think a scope adds much value for us here. Also, for future reference, you don't need to use the `self.all` it is implied. Could just be: `scope :updated_at_desc, -> { order(updated_at: :desc) }`
No need to change this one now though, it works.